src/content/docs/workflows/example-workflows/pre-mfa-workflow.mdx

```diff
@@ -0,0 +1,150 @@
+
+```jsx
+{
+  "request": {
+  },
+  "context": {
+    "auth": {
+			"connectionId": "conn_01945d3ccf4926118bfdeb6e1158edb5" // connection ID the user auth'd with
+		},
+	  "domains": {
+	    "kindeDomain": "https://example.kinde.com"
+	  },
+    "mfa": {
+      "policy": "required", // required | optional | off
+      "context": "environment", // environment or organization
+      "enabled_factors": ["mfa:sms", "mfa:email", "mfa:authenticator_app"], // factors you have enabled for this context
+      "is_user_role_exempt": false, // advanced orgs can make specific roles exempt from MFA
+      "is_user_connection_exempt": false // advanced orgs can make enterprise connections exempt from MFA
+    }
+    "user": { 
+	    "id": "kp_77d28fc7b16240dd9ec12b08071fe46e" // the users ID
+	  },
+    "workflow": {
+	    "trigger": "user:pre_mfa"
+	  },
+    "application": {
+	    "clientId": "cee9743fc7ee4d2e84061fe1a662051d"
+	   },
+    "organization": {
+	    "code": "org_75ad9f26d2c"
+	   }
+  }
+}
+```
+
+### Workflow settings
+
+```jsx
+export const workflowSettings = {
+  id: "preMFA",
+  name: "Pre-MFA",
+  failurePolicy: {
+    action: "stop",
+  },
+  trigger: "user:pre_mfa",
+  bindings: {
+    "kinde.mfa": {}, // required to change the MFA policy
+  },
+};
+```
+
+### MFA binding
+
+The `kinde.mfa` binding is used to modify the MFA policy for the current auth flow.
+
+```jsx
+// In the current flow the user will be asked to complete MFA
+kinde.mfa.setEnforcementPolicy("required");
+
+// In the current flow the user will be exempt from MFA
+kinde.mfa.setEnforcementPolicy("skip");
+```
+
+### External system query example
+
+This example demonstrates using the [environment variables](/build/env-variables/add-manage-env-variables/) feature to store a secret API_KEY to call a FGA system, to determine if a user needs to do MFA.
+
+```tsx
+import {
+  onPreMFAEvent,
+  getKindeAccessTokenHandle,
+  WorkflowSettings,
+  WorkflowTrigger,
+  createKindeAPI
+} from "@kinde/infrastructure";
+
+export const workflowSettings: WorkflowSettings = {
+  id: "preMFA",
+  name: "Pre MFA",
+  failurePolicy: {
+    action: "stop",
+  },
+  trigger: WorkflowTrigger.PreMFA,
+  bindings: {
+    "kinde.fetch": {}, // Required for external API calls
+    "kinde.env": {}, // required to access your environment variables
+    url: {}, // required for url params
+  },
+};
+
+export default async function handlePreMFA(event: onPreMFAEvent) {
+
+	const FGA_API_KEY = getEnvironmentVariable('MY_FGA_API_KEY')?.value
+	const {user, organization} = event.context;
+  const { data: fgaData } = await fetch(`https://someFGAsystem.io/?api_key=${FGA_API_KEY}`, {
+    method: "POST",
+    responseFormat: 'json',
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: new URLSearchParams({
+	    userId: user.id,
+	    orgCode: organization.code
+    })
+  });
+  
+  const isMfaRequired = fgaData.some(permission => permission === 'delete:sensitive')l
```

Comment:
@clairekinde11 this `'delete:sensitive')l` should be `'delete:sensitive');` a semi colon at the end, I made a typo